tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I'm not sure -- maybe natural_exponential_decay to more strongly indicate that it's related to exp ?